src/user/library/index.ts

```diff
@@ -86,6 +84,7 @@ export async function transformNewForm(
     type: form.type,
     budget: form.budget,
     adSets: transformedAdSet,
+    paymentType: form.paymentType.toUpperCase() as PaymentType,
```

Comment:
Even if I uppercase on backend, this failed. So that is why it gets transformed here. Any thoughts @tackley?